src/main/java/telraam/logic/positioner/Position.java

```diff
@@ -0,0 +1,19 @@
+package telraam.logic.positioner;
+
+import lombok.Getter;
+import lombok.Setter;
+import telraam.database.models.Team;
+import telraam.websocket.WebSocketMessageSingleton;
+
+@Getter @Setter
+public class Position {
+    private int teamId;
+    private float progress; // Progress of the lap. Between 0-1
+    private float speed; // Current speed. Progress / second
+
+    public Position(int teamId) {
+        this.teamId = teamId;
+        this.progress = 0;
+        this.speed = 0;
+    }
+}
```

Comment:
I would do this like this

```suggestion
@Getter @Setter
public class Position {
    private int teamId;
    private float progress = 0; // Progress of the lap. Between 0-1
    private float speed = 0; // Current speed. Progress / second

    public Position(int teamId) {
        this.teamId = teamId;
    }
}
```